views/home.hbs

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="/css/vendor/foundation.min.css">
     <script type="text/javascript" src="/js/vendor/jquery.js" defer></script>
     <script type="text/javascript" src="/js/vendor/foundation.min.js" defer></script>
+    <script type="text/javascript" src="/js/test.js" defer></script>
```

Comment:
If we have 3 `<script src="..." defer>` tags in the `<head>`, are we guaranteed they will load in a predictable order? Or is it possible that "/js/test.js" can load before "/js/vendor/jquery.js" and calling `$(document).foundation()` in "/js/test.js" could fail (since neither jQuery or Foundation are loaded)?

Reply:
[Per MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script), yes: "Scripts with the defer attribute will execute in the order in which they appear in the document."